posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+
+![](images/clinical-trial-process.png)
+
+1.  **Collection** – Initial data handling using:
+
+    -   [**tidyverse**](https://www.tidyverse.org/) for general data manipulation.
+
+    -   [**diffdf**](https://cran.r-project.org/web/packages/diffdf/vignettes/diffdf-basic.html) for tracking differences in datasets.
+
+    -   [**metatools**](https://pharmaverse.github.io/metatools/) for metadata management and transformation.
+
+2.  **Tabulation** – Structuring and validating data with:
+
+    -   [**thevalidatoR**](https://github.com/marketplace/actions/r-package-validation-report) for data validation checks.
+
+    -   [**covtracer**](https://github.com/Genentech/covtracer) for ensuring code traceability and reproducibility.
+
+3.  **Analysis Ready** – Preparing data for regulatory-grade analytics:
+
+    -   [**admiral**](https://github.com/pharmaverse/admiral) for standardized data derivations.
+
+    -   [**metacore**](https://cran.r-project.org/web/packages/metacore/index.html) for metadata-driven structures.
+
+    -   [**dplyr**](https://dplyr.tidyverse.org/) (part of the tidyverse) for data manipulation
+
+4.  **Analysis Results** – Incorporating visualization and modeling tools:
+
+    -   [**ggplot2**](https://ggplot2.tidyverse.org/) (part of the tidyverse) for advanced data visualization.
+
+    -   [**visR**](https://github.com/openpharma/visR) for survival analysis and Kaplan-Meier plots.
+
+    -   [**pharmaRTF**](https://cran.r-project.org/web/packages/pharmaRTF/index.html) for creating submission-ready reports.
+
+    -   [**rtables**](https://github.com/insightsengineering/rtables) for structured table outputs
+
+    -   [**Tplyr**](https://github.com/atorus-research/Tplyr) for clinical summary tables
+
+    -   The pharmaverse offers multiple table-making packages, including [rtables](https://insightsengineering.github.io/rtables/v0.6.10/articles/rtables.html), [chevron](https://insightsengineering.github.io/chevron/latest-tag/), [pharmaRTF](https://atorus-research.github.io/pharmaRTF/), [gtsummary](https://www.danieldsjoberg.com/gtsummary/), [cards](https://insightsengineering.github.io/cards/latest-tag/), [tfrmt](https://gsk-biostatistics.github.io/tfrmt/), [tidytlg](https://pharmaverse.github.io/tidytlg/main/), and [cardinal](https://pharmaverse.github.io/cardinal/). You can find more tools on the [TLGs page](https://pharmaverse.org/e2eclinical/tlg/). Each serves different needs, and it is up to the programmer to choose the best fit.
+
+5.  **Submission** – Final reporting and submission:
+
+    -   [**xportr**](https://github.com/atorus-research/xportr) for CDISC-compliant dataset export.
+
+    -   [**pkglite**](https://merck.github.io/pkglite/) for package management and tracking.
+
+By using these tools, organizations can optimize their data pipeline, ensuring clinical data is well-structured, validated, and ready for regulatory submission with ease.
+
+## **Where Do I Start? Understanding Tasks & Objectives**
```

Comment:
Feels like this section is superfluous because more packages are being listed here when in the section previous to this one, a list was provided for all stages of clinical trial development. i'd remove this section and absorb key content into the previous one.